web/components/tipper.tsx

```diff
@@ -67,30 +59,75 @@ export function Tipper(prop: {
         fromId: user.id,
         toId: comment.userId,
       })
-    }, 1500)
+    }
   )
-  // instant save on unrender
-  useEffect(() => () => void saveTip.flush(), [saveTip])
 
   const addTip = (delta: number) => {
-    setLocalTip(localTip + delta)
```

Comment:
Hmm, did this allow for larger tips in one transaction if you clicked multiple times?

Might want to support that use case, especially if you have to wait a few seconds for one tip to go through.

(A side note: I think the info popup appears over the market tip button! Which makes it hard to tip multiple times rapidly.)

Reply:
I don't think it did that, but yeah that info popup is really annoying in general for mobile, will get rid of in another PR